includes/admin/discounts/edit-discount.php

```diff
@@ -177,31 +177,19 @@
                     </th>
                     <td>
                         <input name="start_date" id="edd-start" type="text" value="<?php echo esc_attr( false !== $discount->start_date ? $start_date : '' ); ?>" class="edd_datepicker" data-format="<?php echo esc_attr( edd_get_date_picker_format() ); ?>" placeholder="<?php echo esc_attr( edd_get_date_picker_format() ); ?>" />
-						<?php
-						echo EDD()->html->select( array(
-							'name'             => 'start_date_hour',
-							'options'          => $hours,
-							'selected'         => false !== $discount->start_date ? $start_hour : '0',
-							'chosen'           => true,
-							'class'            => 'edd-time',
-							'show_option_none' => false,
-							'show_option_all'  => false
-						) );
-						?>
-						:
-						<?php
-						echo EDD()->html->select( array(
-							'name'             => 'start_date_minute',
-							'options'          => $minutes,
-							'selected'         => false !== $discount->start_date ? $start_minute : '0',
-							'chosen'           => true,
-							'class'            => 'edd-time',
-							'show_option_none' => false,
-							'show_option_all'  => false
-						) );
-
-						echo esc_html( ' (' . edd_get_timezone_abbr() . ')' );
-						?>
+
+												<label class="screen-reader-text" for="start-date-hour">
```

Comment:
Funky spacing here.

Reply:
@ashleyfae Whole file seems funky: https://github.com/easydigitaldownloads/easy-digital-downloads/blob/release/3.0/includes/admin/discounts/edit-discount.php

Should I format it all?

Reply:
@spencerfinnell Yeah, go for it.